database restore fails on Android 6 (for current dev version)

**Describe the bug:**
Tried to restore a c:geo database on an Android 6 system with current dev. version. c:geo stops without any recognizable error message.

After some debugging an error message appears in the log:

```
Caused by: java.lang.NoSuchMethodError: No virtual method toPath()Ljava/nio/file/Path; in class Ljava/io/File; or its super classes (declaration of 'java.io.File' appears in /system/framework/core-libart.jar)
        at org.apache.commons.io.FileUtils.doCopyFile(FileUtils.java:1303)
        at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:797)
        at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:747)
        at cgeo.geocaching.utils.FileUtils.copy(FileUtils.java:205)
```

Downgrading to apache.commons.io Version 2.5 helps. Have not tested 2.6 yet. 
Could not find any dependencies to Android SDK mentioned on apache commons website at a quick glance, needs to be investigated further.

@bekuno, can you have a look at this?

**To Reproduce:**
- Use a device running Android 6 (in my case: a physical device running Android 6.0.1)
- Make a database backup with a release version of c:geo (in my case: 2020.03.30)
- Install current dev version, accept requested rights, and try to restore the database backup

**Version of c:geo used:**
- DB backup created with 2020.03.30 release version
- Trying to restore backup with current master branch dev version

**Is the problem reproducible:**
Yes

--- a/build.gradle
+++ b/build.gradle
@@ -95,6 +95,12 @@ allprojects {
                         if (selection.candidate.group == 'org.assertj' && selection.candidate.module == 'assertj-core' && selection.candidate.version.substring(0,1).toInteger() > 2) {
                             selection.reject("assertj 3.x or higher requires Java 7 SE classes not available in Android")
                         }
+
+                        // Apache Commons IO 2.6 fails in FileUtils.deleteDirectory()
+                        // (uses java.nio.file.Path - which is first contained in Android API26)
+                        if (selection.candidate.group == 'commons-io' && selection.candidate.module == 'commons-io' && selection.candidate.version.substring(0,1).toInteger()*10+selection.candidate.version.substring(2,3).toInteger() > 25) {
+                            selection.reject("Apache Commons IO 2.6 calls methods not available in the Android runtime untli API26.")
+                        }
                     }
                 }
                 // new versions of checkstyle use guava 27, which has troublesome dependencies: https://groups.google.com/forum/#!topic/guava-announce/Km82fZG68Sw

--- a/main/build.gradle
+++ b/main/build.gradle
@@ -246,9 +246,12 @@ dependencies {
     // Apache Commons
     implementation 'org.apache.commons:commons-collections4:4.4'
     implementation 'org.apache.commons:commons-compress:1.20'
-    implementation 'commons-io:commons-io:2.7'
     implementation 'org.apache.commons:commons-lang3:3.10'
     implementation 'org.apache.commons:commons-text:1.8'
+    // commons-io 2.6 uses java.nio.file.Path - which is first contained in Android API26
+    // could be replaced by Storage Access Framework
+    //noinspection GradleDependency
+    implementation 'commons-io:commons-io:2.5'
 
     // AssertJ for testing, needed both for local unit tests and Android instrumentation tests
     def assertJVersion = '2.9.1'

